iina/PlaylistViewController.swift

```diff
@@ -436,6 +440,7 @@ class PlaylistViewController: NSViewController, NSTableViewDataSource, NSTableVi
     let tv = notification.object as! NSTableView
     if tv == playlistTableView {
       showTotalLength()
+      selectedRows = playlistTableView.selectedRowIndexes
```

Comment:
`selectedRows` is currently only used by the context menu. Is this line necessary?

Reply:
No.